tests/newman/tests/data-api.json

```diff
@@ -19,7 +19,8 @@
 									"script": {
 										"id": "ff0e67a8-97f3-414d-a136-e46252cfbacf",
 										"exec": [
-											""
+											"// Skipping timing out requests -- EES-234",
```

Comment:
Do you have plans to revisit these?  I'm just worried about them getting forgotten about, so would it be worth raising a ticket in the backlog to revisit when they're not timing out anymore?

Reply:
I've left a comment on EES-234 for them to be uncommented during/after the Data API performance review: https://alpha-dfe-data.atlassian.net/browse/EES-234